index.html

```diff
@@ -8,9 +8,39 @@
       href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.css"
     />
     <link rel="stylesheet" type="text/css" href="style.css" />
+    <link href="https://fonts.googleapis.com/css?family=Playfair+Display&display=swap" rel="stylesheet">
   </head>
   <body>
-    <div class="js-grid-container grid-container"></div>
+    <header class="header">Etch a sketch</header>
+    <div class="js-radio-buttons section-button">
+      <form>
+        <input
+          class="button__radio"
+          type="radio"
+          name="color"
+          value="black"
+          checked
+        />
+        Black
+        <input
+          class="button__radio"
+          type="radio"
+          name="color"
+          value="random"
+        />
+        Random RGB
+        <input
+          class="button__radio"
+          type="radio"
+          name="color"
+          value="gradient"
+        />
+        Gradient Black</form></br>
+      </form>
+      <button class="js-resize-grid button__submit" type="submit">Resize grid</button>
```

Comment:
- Use camel case for the button text.
- Use `type="button"` by default. `type="submit"` is used when you want to submit a form.

Reply:
I don't understand why I need to use camel case for the button text.